packages/fast-viewer-react/webpack.config.js

```diff
@@ -1,50 +1,54 @@
-const webpack = require("webpack");
 const path = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 const appDir = path.resolve(__dirname, "./app");
 const outDir = path.resolve(__dirname, "./www");
 
-module.exports = {
-    devtool: "inline-source-map",
-    entry: path.resolve(appDir, "index.tsx"),
-    output: {
-        path: path.resolve("./www"),
-        publicPath: "/",
-        filename: "app.js"
-    },
-    mode: process.env.NODE_ENV || "development",
-    resolve: {
-        extensions: [".js", ".ts", ".tsx"],
-    },
-    module: {
-        rules: [
-            {
-                test: /\.tsx?$/,
-                use: [
-                    {
-                        loader: "ts-loader",
-                        options: {
-                            compilerOptions: {
-                                declaration: false,
+module.exports = (env, args) => {
+
+    const isProduction = args.mode === "production";
+
+    return {
+        devServer: {
+            compress: false,
+            historyApiFallback: true,
+            open: true,
+            port: 3000
+        },
```

Comment:
Why are we moving all the keys around and changing this to be a function?

Reply:
I followed the optimizations put in place for inline-source-maps by Nick.  I did alphabetize the properties to cleanup and make it easier to find things while I was in there.   I mentioned in the task description the changes I made.

Reply:
please remove these changes as they are not related to the security vulnerability indicated by the PR title which is the only thing that will be recorded in git history, we can have a separate PR that updates all the webpack configs, I think that would make the most sense.